src/input-group/input-group.style.tsx

```diff
@@ -143,4 +124,18 @@ export const AddOnContainer = styled.div<AddonStyleProps>`
             `;
         }
     }}
+
```

Comment:
need to pass the position prop to AddOnContainer from `input-group.tsx`

Reply:
Lol thanks for catching. I wonder how did i miss it. :/